AWS/legos/aws_list_attached_user_policies/README.md

```diff
@@ -0,0 +1,25 @@
+[<img align="left" src="https://unskript.com/assets/favicon.png" width="100" height="100" style="padding-right: 5px">](https://unskript.com/assets/favicon.png) 
+<h1>AWS List Attached User Policies </h1>
+
+## Description
+This Lego AWS list Attached User Policies and gives a list of policies.

```

Comment:
Please change the description to:

This Lego returns the list of Policies attached to a User.